packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -52,6 +53,10 @@ export class WaferMap extends FoundationElement {
     @attr({
         attribute: 'color-scale-mode'
     })
+    public readonly canvas!: HTMLCanvasElement;
+
+    public readonly svgRoot!: HTMLElement;
```

Comment:
Can you mark these as internal, ie:
```suggestion
    /**
     * @internal
     */
    public readonly canvas!: HTMLCanvasElement;

    /**
     * @internal
     */
    public readonly svgRoot!: HTMLElement;
```